gammapy/modeling/models/spatial.py

```diff
@@ -1036,6 +1049,10 @@ def __init__(
         self.filename = filename
         super().__init__()
 
+    @property
+    def map(self):
+        return self._map
```

Comment:
Add one line docstring and return type here.

Reply:
done